### ISSUE TITLE
Rename to make clear it is not an addin

### ISSUE THREAD

**Issue by pascalberger** [asmagin/Cake.Sitecore#48]
As this package contains scripts and tools and is not a Cake addin (e.g. doesn't provide any aliases), I would suggest to rename the package to `Cake.Sitecore.Recipe`, which is used by other similar packages (e.g. [Cake.Recipe](https://www.nuget.org/packages/Cake.Recipe/), [Cake.Wyam.Recipe](https://www.nuget.org/packages/Cake.Wyam.Recipe/) or [Cake.Issues.Recipe](https://www.nuget.org/packages/Cake.Issues.Recipe/)). While naming packages like this is not technical necessary this convention adopted by the community helps people understand what the content of a package is.

**Comment by asmagin** [comment]
I do not think that it will make sense to do right now with the current version, as it used by many projects.
However, your feedback makes sense and if we will see that we need any significant overhaul with breaking changes, we can incorporate it in the next major version.
